src/sentry/api/serializers/models/project.py

```diff
@@ -269,6 +269,20 @@ def get_attrs(self, item_list, user):
         attrs = super(ProjectSummarySerializer,
                       self).get_attrs(item_list, user)
 
+        project_envs = EnvironmentProject.objects.filter(
```

Comment:
Two `exclude` statements here since `is_hidden` is a `models.NullBooleanField` meaning it can accept `False` and `None` as values. So excluding `True` was the cleanest method here. And putting two conditions in the same `exclude` call ANDs the conditions rather than OR.

Reply:
This seems totally reasonable. Thx @lynnagara for the heads up.